agent/rpm/exclude.pats

```diff
@@ -6,3 +6,7 @@ unittests
 tests/
 mock-bin/
 util-scripts/configs/
+ansible/bin
```

Comment:
Given these patterns, what should be included from the `ansible` subtree in the RPM?

Reply:
AFAICT, the only thing needed is ansible/ara/.

Reply:
Removed the need for `exclude.pats` entirely by installing only the pieces we needed from the ansible tree.